MAUI/DigitalGauge/Customize-character-segments.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: Customize Character Segments in Maui Digital Gauge | Syncfusion
+description: Learn here all about Customize Character Segments support in Syncfusion Maui Digital Gauge (SfDigitalGauge) control and more.
+platform: maui 
+control: DigitalGauge
+documentation: ug
+---
+
+# Customize Character Segments in Maui Digital Gauge (SfDigitalGauge)
+
+The characters of a digital gauge can be customized in terms of `CharacterWidth`, `CharacterHeight`, `CharacterSpacing`, and `CharacterStrokeColor`.
+
+## customize character size
+
+The values of digital characters are scaled by altering the height and width of digital characters. It is achieved by setting the `CharacterHeight`and `CharacterWidth` properties in the digital gauge. Default value of `CharacterHeight` and `CharacterWidth` is 25.
```

Comment:
The values of digital characters are scaled by altering the height and width of digital characters. This is achieved by setting the `CharacterHeight` and `CharacterWidth` properties in the digital gauge. The default value of `CharacterHeight` and `CharacterWidth` is 25.